test/controllers/homeworks_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
```

Comment:
Good work on adding tests. Generally for tests we can assume that Rails will work as advertised, so try to make tests just for logic you have added, such as an input validation.